broskichat-app/src/App.test.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import Header from './Components/Partials/Header';
+
+import TestRenderer from 'react-test-renderer';
 
 it('renders without crashing', () => {
   const div = document.createElement('div');
   ReactDOM.render(<App />, div);
   ReactDOM.unmountComponentAtNode(div);
 });
+
+describe('<Header />', () => {
```

Comment:
@akeon-lee nice job adding a test assertion here! We'll probably move these to individual files in the future if it gets too long.